### PR TITLE
Add reusable workflows on DEV

### PR DESCRIPTION
**Pull request by DmitriyMolch**

### CODE REVIEW

**Review by krolol — APPROVED** [review]

File: .github/workflows/prod.yaml [hunk 1]
```diff
@@ -5,58 +5,18 @@ on:
...
+    uses: Cerebellum-Network/reusable-workflows/.github/workflows/build-and-upload-static.yaml@1.0.0
+    with:
+      runs-on: '["self-hosted", "cere-network-large-workers"]'
+      build_container: 'node:14-buster'
+      deploy_container: 'ubuntu:20.04'
+      install_packages_command: 'yarn install'
+      build_command: 'yarn build'
+      path_to_static_files_to_upload: 'build'
+    secrets:
+      NETWORK_AWS_ACCESS_KEY_ID: ${{ secrets.DEV_NETWORK_AWS_ACCESS_KEY_ID }}
```
Comment (krolol) [inline .github/workflows/prod.yaml:18]: Need to change to PROD secrets

File: .github/workflows/stage.yaml [hunk 2]
```diff
@@ -6,58 +6,18 @@ on:
...
+    uses: Cerebellum-Network/reusable-workflows/.github/workflows/build-and-upload-static.yaml@1.0.0
+    with:
+      runs-on: '["self-hosted", "cere-network-large-workers"]'
+      build_container: 'node:14-buster'
+      deploy_container: 'ubuntu:20.04'
+      install_packages_command: 'yarn install'
+      build_command: 'yarn build'
+      path_to_static_files_to_upload: 'build'
+    secrets:
+      NETWORK_AWS_ACCESS_KEY_ID: ${{ secrets.DEV_NETWORK_AWS_ACCESS_KEY_ID }}
```
Comment (krolol) [inline .github/workflows/stage.yaml:19]: Need to change to STAGE secrets

**Review by krolol — CHANGES_REQUESTED** [review]

File: .github/workflows/prod.yaml (hunk 1 above)
Comment (DmitriyMolch) [reply .github/workflows/prod.yaml]: Fixed

File: .github/workflows/stage.yaml (hunk 2 above)
Comment (DmitriyMolch) [reply .github/workflows/stage.yaml]: Fixed

**Review by AndreiNavoichyk — APPROVED** [review]

**Review by krolol — APPROVED** [review]

**Pull request merged**